core/src/main/java/io/grpc/util/RoundRobinLoadBalancerFactory.java

```diff
@@ -76,15 +86,17 @@ public LoadBalancer newLoadBalancer(LoadBalancer.Helper helper) {
   }
 
   @VisibleForTesting
-  static class RoundRobinLoadBalancer extends LoadBalancer {
+  static final class RoundRobinLoadBalancer extends LoadBalancer {
+    @VisibleForTesting
+    static final Attributes.Key<Ref<ConnectivityStateInfo>> STATE_INFO =
+        Attributes.Key.of("state-info");
+    private static final Attributes.Key<Ref<EquivalentAddressGroup>> ADDRESS_GROUP =
```

Comment:
You don't need this, because `Subchannel.getAddresses()` is exactly what you need.

Reply:
Done